frontend/src/components/webflow/WebflowPage.tsx

```diff
@@ -1,36 +1,40 @@
-import parseHtml from "html-react-parser";
-import Head from "next/head";
 import React, { useContext } from "react";
 import getTexts from "../../../public/texts/texts";
 import UserContext from "../context/UserContext";
 import WideLayout from "../layouts/WideLayout";
+import { DevLinkProvider } from "../../../devlink/DevLinkProvider";
 
 export default function WebflowPage({
-  bodyContent,
-  headContent,
-  pageKey,
-  className,
-  hideFooter,
-  noHeader,
+  children,
   title,
+  pageKey,
   description,
+  isStaticPage = true,
+  transparentHeader,
+  isHubPage,
+  hubUrl,
+  isLandingPage,
+  showSuffix,
+  isLocationHub,
 }: any) {
```

Comment:
Doesn't this break all other places where use still use this component with the old properties? We will still need the old component for all the places where we use static webflow pages until we convert them all to devlink which I'd do in a seperate PR if at all